README.md

```diff
@@ -5,47 +5,134 @@
+| User-Defined Functions (UDFs)    | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/howto/udfs-in-pixeltable.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| Comparing Object Detection Models | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/tutorials/object-detection-in-videos.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| Experimenting with Chunking (RAG) | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/tutorials/rag-operations.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>
+| Working with External Files    | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/howto/working-with-external-files.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| Integrationg with Label Studio for Annotations    | <a target="_blank" href="https://pixeltable.readme.io/docs/label-studio"> <img src="https://img.shields.io/badge/Docs-Label Studio-blue" alt="Visit our documentation"/></a>
 
+## 🧱 Code Samples
+
+### All media data (videos, images, audio) resides in external files, and Pixeltable stores references to those
 ```python
-import pixeltable as pxt
-pxt.init()
+# The files can be local or remote (e.g., in S3). For the latter, Pixeltable automatically caches the files locally on access.
+v = pxt.create_table('external_data.videos', {'video': pxt.VideoType()})
+
+prefix = 's3://multimedia-commons/'
+paths = [
+    'data/videos/mp4/ffe/ffb/ffeffbef41bbc269810b2a1a888de.mp4',
+    'data/videos/mp4/ffe/feb/ffefebb41485539f964760e6115fbc44.mp4',
+    'data/videos/mp4/ffe/f73/ffef7384d698b5f70d411c696247169.mp4'
+]
+v.insert({'video': prefix + p} for p in paths)
 ```
-> [!NOTE]
-> Check out the [Pixeltable Basics](https://pixeltable.readme.io/docs/pixeltable-basics) tutorial for a tour of its most important features.
+Learn more on how to [access the original file paths and work with your data](https://pixeltable.readme.io/docs/working-with-external-files).
 
-## 💡 Get Started
-Learn how to create tables, populate them with data, and enhance them with built-in or user-defined transformations and AI operations.
+### Create a computed column based on a function calling an object detection model
+```python
+# Pixeltable is downloading the model from Huggingface (if necessary), instantiating it, and caching it for later use
```

Comment:
Not sure this comment is helpful here.

Also, add `from pixeltable.functions import huggingface`

Reply:
Same as above. I left pip install and import statement out of these code snippets as even with those and they are not runnable by themselves and the goal is to focus the attention on the logic and the simplicity/flexibility of the code.

Reply:
Ok, but can we get rid of the comment? I don't think it adds anything (it's cargo-culted from the notebook but I think it's confusing out of context)